### PR TITLE
Utilise shared ruboconfig

### PR DESCRIPTION
**Pull request by GeorgeA93**
GitHub pointed this out, which spurred me to update rubocop and use our shared config!

screen shot 2017-12-13 at 14 13 09

I am unsure why this is trying to build on Circle. From what I can see it hasn't built on Circle for the last three years, and most other pull requests don't even have build statuses...!

### CODE REVIEW

File: ibandit.gemspec [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-                    'national banking details; and validate an IBAN\'s check ' \
-                    'digits and format.'
-  gem.email = ['grey@gocardless.com']
+  gem.authors = ["Grey Baker"]
+  gem.description = "Ibandit is a Ruby library for manipulating and " \
+                    "validating IBANs. It allows you to construct an IBAN " \
+                    "from national banking details; deconstruct an IBAN into " \
+                    "national banking details; and validate an IBAN's check " \
+                    "digits and format."
+  gem.email = ["grey@gocardless.com"]
```
Comment (nickcampbell18) [inline ibandit.gemspec]: We should update this to `developers@gocardless.com`, like https://github.com/gocardless/coach/blob/master/coach.gemspec#L11.

Sorry @greysteil!

Comment (greysteil) [reply ibandit.gemspec]: Makes sense to me - I don't even have push access to this anymore!

File: .travis.yml [hunk 2]
```diff
@@ -4,9 +4,6 @@ matrix:
   allow_failures:
 
 rvm:
-  - 2.1.10
-  - 2.2.7
-  - 2.3.4
```
Comment (ivgiuliani) [inline .travis.yml]: Drive-by comment: do we have a policy around deprecating major ruby versions in open source projects? I recall seeing some discussions around this a while ago (for statesman I think)

Comment (GeorgeA93) [reply .travis.yml]: Unsure of any policy we may have. I'm pretty sure that these versions never worked. @jaysh can confirm

Comment (jaysh) [reply .travis.yml]: @ivgiuliani We should double check the next time we plan on deprecating (to be clear, this PR is not actually deprecating anything - it's already broken).

In this instance, the failure here was that the code doesn't work at all with Ruby < 2.4  and we weren't observing our builds so this went unnoticed. Now that we have visible build results in the PRs, this shouldn't happen going forwards. That said, if you feel there's value in getting the project working with Ruby 2.1+ again, I can put a ticket in for it.

File: spec/ibandit/structure_spec.rb [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+
+describe "structures.yml" do
+  FILE = File.expand_path("../../../data/structures.yml", __FILE__)
   STRUCTURES = YAML.load_file(FILE)
 
   STRUCTURES.each do |country, rules|
     context country do
       rules.each do |rule, value|
-        next unless rule =~ /_format$/
+        next unless rule.match?(/_format$/)
```
Comment (jaysh) [inline spec/ibandit/structure_spec.rb]: Leaving a note as discussed to change this back.

File: .travis.yml (hunk 2 above)
Comment (ivgiuliani) [reply .travis.yml]: For reference, this is what I was referring to: https://github.com/gocardless/statesman/pull/253
Not saying we should introduce the same policy here (though in this case it's simpler since we only care about the ruby version, not rails too), but worth thinking about it.

**Comment by jaysh** [comment]
Done in #85
